modules/installation-configuration-parameters.adoc

```diff
@@ -2689,6 +2759,35 @@ Additional VMware vSphere configuration parameters are described in the followin
       datacenters:
 |Lists and defines the datacenters where {product-title} virtual machines (VMs) operate. The list of datacenters must match the list of datacenters specified in the `failureDomains` field.
 |String
+
+|platform:
+  vsphere:
+    vcenters:
+      password:
```

Comment:
each vCenter is a separate entry in the vcenters slice. this allows multiple vCenters to be defined potentially(though its not supported at the moment).
```suggestion
      - password: <pw>
        port: 443
        server: <vcenter fqdn>
```

Reply:
That makes total sense. Each param is depicted this way because the install writers changed the old config format in the reference tables (e.g. `platform.vsphere.vcenters.password`) to this newer one that saves horizontal space. We're relying on the description/values column to convey things like what's optional and what needs to go together as a set. 

For example, for the compute section of the install-config, instead of documenting `architecture`, `name`, `platform`, etc like this, we rely on the value field of `compute.architecture` to describe the structure: "[Array of MachinePool objects](https://docs.openshift.com/container-platform/4.14/installing/installing_vsphere/installation-config-parameters-vsphere.html#installation-configuration-parameters-optional_installation-config-parameters-vsphere:~:text=the%20compute%20nodes.-,Array%20of%20MachinePool%20objects.,-compute%3A%0A%20%20architecture%3A)"